### PR TITLE
add stopSpeaking api to tts

### PR DESCRIPTION
**Pull request by noelweichbrodt**
The hardest part was figuring out how to handle the additional button in the example project.

### CODE REVIEW

File: Spokestack/TextToSpeech.swift [hunk 1]
```diff
@@ -111,6 +111,12 @@ private let apiQueue = DispatchQueue(label: TTSQueueName, qos: .userInitiated, a
         self.synthesize(input: input, success: play)
     }
     
+    /// Stops playback of the current synthesi result.
```
Comment (space-pope) [inline Spokestack/TextToSpeech.swift:114]:
```suggestion
    /// Stops playback of the current synthesis result.
```

**Review by space-pope — COMMENTED** [review]
LGTM, with one question:

If a client calls `speak` twice in quick succession, is the audio for the second tts result queued? If so, does `stopSpeaking` have any implications for queued audio?

**Comment by noelweichbrodt** [comment]
per line 108, each call to `speak` will replace Spokestack's `AVPlayer.currentItem` with the new synthesis result. It maintains no queue of results.

**Review by space-pope — APPROVED** [review]
Hm, I don't think that's necessarily the best approach for TTS (imagine a long prompt followed by a short one), but not an issue for this PR either way.

**Pull request merged**